subsys/bluetooth/mesh/gatt_services.c

```diff
@@ -91,6 +97,7 @@ static struct bt_mesh_proxy_client {
 		BLACKLIST,
 		PROV,
 	} filter_type;
+	uint8_t gatt_type;
```

Comment:
This is in preparation for the `MESH_GATT_` enum to get a `MESH_GATT_PROXY_CLIENT` as well, right?